Update docs with API key details (#11593)
PostHog/posthog.com: contents/docs/cdp/sources/stripe.md

The Stripe connector can link charges, customers, invoices, prices, products, subscriptions, and balance transactions to PostHog. 

> ## Creating a Stripe API key

Reviewer: ```suggestion
## Creating a Stripe API key

```

> You need a Stripe API key to create a connector. Head to your Strip dashboard > **Developers** > **API keys**, under **Restricted keys**, click [+ Create a restricted key](https://dashboard.stripe.com/apikeys/create).

Reviewer: ```suggestion
You need a Stripe API key to create a connector. Head to your Stripe dashboard > **Developers** > **API keys**, under **Restricted keys**, click [+ Create a restricted key](https://dashboard.stripe.com/apikeys/create).
```

> You will need to give your API key the following **Read** permissions.

Reviewer: ```suggestion
You need to give your API key the following **Read** permissions:
```

| Resource Type | Required Read Permissions                                |
|--------------|--------------------------------------------------------|
| Core         | Balance transaction sources, Charges, Customer, Product  |
| Billing      | Invoice, Price, Subscription                            |
| Connected    | All resources                                           |

## Adding a data source 

1. In PostHog, go to the [Data pipeline page](https://us.posthog.com/pipeline/sources) select the **Sources** tab.
2. Click **+ New source** button and select Stripe by clicking the **Link** button.

> 3. Get your Account ID from your Stripe's **Settings** > **Business**, select the [Account details](https://dashboard.stripe.com/settings/account) tab. Click your **Account ID** or press `⌘` + `I` to copy your ID.

Reviewer: ```suggestion
3. In Stripe, get your Account ID by going to **Settings** > **Business**, selecting the [Account details](https://dashboard.stripe.com/settings/account) tab, and clicking your **Account ID** or pressing `⌘` + `I` to copy your ID.
```
